lib/git-shell-out-strategy.js

```diff
@@ -270,6 +270,9 @@ export default class GitShellOutStrategy {
             } else if (signal) {
               summary += `exit signal: ${signal}\n`;
             }
+            if (stdin && stdin.length !== 0) {
```

Comment:
It is useful for tracking down broken patches because they're passed to git on stdin, but really it's useful for debugging for any git command that we interact with on stdin. Just patching a shortcoming of the surrounding diagnostic code that I happened to hit here.